### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by sourabhx003** [BenYavor/Wasserstein-GAN-Tensorflow-2#1]
Cant find Wasserstein Loss anywhere
